src/aura/CRLP_Tooltip/CRLP_Tooltip.cmp

```diff
@@ -60,7 +60,7 @@
                 <lightning:icon class="slds-icon-text-default" size="xx-small"
                                 iconName="utility:info" alternativeText="Help"/>
             </button>
-            <div aura:id="divSelectHelp" id="divHelp" class="slds-popover slds-popover_tooltip slds-nubbin_top-left slds-popover_small slds-fall-into-ground" role="tooltip" aria-live="polite">
+            <div aura:id="divSelectHelp" id="divHelp" class="slds-popover slds-popover_tooltip slds-nubbin_top-left slds-popover_medium slds-fall-into-ground" role="tooltip" aria-live="polite">
```

Comment:
I was using the CRLP_Tooltip at one point as a component, but ended up using the helptext base component since that's working now. I think the base popover class changed, and made it so using this requires popover_medium (popver_small looks TERRIBLE). This would fix an existing issue in CRLPs though, so would love to keep it in! https://foundation.my.salesforce.com/a2x1E000007fQ1oQAE

Reply:
Could we just create a separate branch with that change so we can link it back to that work item?